tests/e2e/support.py

```diff
@@ -45,3 +53,24 @@ def run_oc_command(args):
     except subprocess.CalledProcessError as e:
         print(f"Error executing 'oc {' '.join(args)}': {e}")
         return None
+
+
+def create_local_queue(self, cluster_queue):
+    json = {
+        "apiVersion": "kueue.x-k8s.io/v1beta1",
+        "kind": "LocalQueue",
+        "metadata": {
+            "annotations": {"kueue.x-k8s.io/default-queue": "true"},
+            "namespace": self.namespace,
+            "name": "local-queue",
+        },
+        "spec": {"clusterQueue": cluster_queue},
+    }
+    print("LocalQueue doesn't exists, creating LocalQueue!")
+    self.custom_api.create_namespaced_custom_object(
+        group="kueue.x-k8s.io",
+        namespace=self.namespace,
+        plural="localqueues",
+        version="v1beta1",
+        body=json,
+    )
```

Comment:
We could do with an exception here in case it is not possible to create the local queue